content/guides/code-splitting-import.md

```diff
@@ -144,6 +144,28 @@ module.exports = {
 };
 ```
 
+## `import` imports the entire module namespace
+
+Consider the following two examples:
```

Comment:
The statement "The promise is resolved with the module namespace" from https://github.com/tc39/proposal-dynamic-import/issues/21#issuecomment-258127680 brought me right on track. Please add this at the beginning and link to the spec (2.1.1, step 8.c.iv).